controller/api/destination/watcher/prometheus.go

```diff
@@ -34,6 +34,68 @@ type (
 	}
 )
 
+var (
+	informer_lag_secs_buckets = []float64{
+		0.5,  // 500ms
+		1,    // 1s
+		2.5,  // 2.5s
+		5,    // 5s
+		10,   // 10s
+		25,   // 25s
+		50,   // 50s
+		100,  // 1m 40s
+		250,  // 4m 10s
+		1000, // 16m 40s
+	}
+	endpointsInformerLag = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "endpoints_informer_lag_secs",
```

Comment:
The [convention](https://prometheus.io/docs/practices/naming/) is specifically to use _seconds_
```suggestion
			Name:    "endpoints_informer_lag_seconds",
```